test/00_transform/FFT.cu

```diff
@@ -92,6 +92,50 @@ TYPED_TEST(FFTTestComplexTypes, FFT1D1024C2C)
   MATX_EXIT_HANDLER();
 }
 
+TYPED_TEST(FFTTestComplexNonHalfTypes, FFT1DFWD1024C2C)
+{
+  MATX_ENTER_HANDLER();
+  const index_t fft_dim = 1024;
+  this->pb->template InitAndRunTVGenerator<TypeParam>(
+      "00_transforms", "fft_operators", "fft_1d_fwd", {fft_dim, fft_dim});
+
+  tensor_t<TypeParam, 1> av{{fft_dim}};
+  tensor_t<TypeParam, 1> avo{{fft_dim}};
+  this->pb->NumpyToTensorView(av, "a_in");
+
+  // example-begin fft-1
```

Comment:
You either need to remove this name or give it a unique name since it's colliding with the other example names